docs/src/main/asciidoc/spanner.adoc

```diff
@@ -25,55 +27,294 @@ dependencies {
 }
 ----
 
+This setup takes care of bringing in the latest compatible version of Cloud Java Spanner libraries as well.
+
+
+=== Configuration
+
+To setup Spring Data Spanner, you have to configure the following:
+
+* setup the connection details to Google Cloud Spanner
```

Comment:
this bullet doesn't have a period at the end, but the next one does.

some people say only sentences should have periods, but as long as you're consistent i'm ok